README.md

```diff
@@ -261,17 +271,17 @@ sourceFile2.py.zip
 sourceFolder2.zip
 ```
 
-*NB:* When zipping a folder, the root of the archive will contain the contents
-of that folder, not the folder by itself.
+Both the `hash` and `zip` flag can be used together. This will yield the same
+result as having just the `zip` flag, except all file paths will be
+prefixed with the hash of their contents.
+
+*NB:* When zipping a folder, the structure of the folder is flattened since AWS
+Lambda can only detect source files in the root of an archive.
```

Comment:
When I was testing the zip files I could only get Lambda to find the source file if it was in the root of the archive, but apparently you can [specify source files in directories.](https://forums.aws.amazon.com/message.jspa?messageID=682011)